src/definitions/modules/slider.js

```diff
@@ -411,39 +411,43 @@
                         }
                         if (first || module.is.focused()) {
                             var step = module.determine.keyMovement(event);
-                            if (step != NO_STEP) {
+                            if (step !== NO_STEP) {
                                 event.preventDefault();
                                 switch (step) {
-                                    case SINGLE_STEP:
+                                    case SINGLE_STEP: {
```

Comment:
Why is that extra brace encapsulation necessary? (same for all following cases)

Reply:
originally I turned off this rule, but later I decised wrapping it in a block/curly brackets is better to go with the recommended config - it is more readable for larged blocks, also, it should limit the scope of declared variables inside the block

Reply:
I dont agree. Stating better readability can be used almost everywhere 🙂 . We already have the 4 space indent. This is pretty much improved readability already. And there is no need for scoping here as no extra variabled are declared

Reply:
I have resolved other threads for this topic, it is not an issue, it is a CS implied by recommended eslint config.

in https://github.com/fomantic/Fomantic-UI/pull/2590/commits/86018ee62b21a6ad9bf7c31bacd6ccdae61080f7 the whole impact can be seen

we can of course disable it globally, but I actually prefer to keep code blocks wrapped in brackets, which is more readable as start/close bracket can be easily located

Reply:
ok, convinced